src/main/scala/io/sdkman/changelogs/java/MandrelMigrations.scala

```diff
@@ -60,4 +60,58 @@ class MandrelMigrations {
       )
     ).validate()
       .insert()
+
+  @ChangeSet(
```

Comment:
You can do something like this

https://github.com/sdkman/sdkman-db-migrations/blob/master/src/main/scala/io/sdkman/changelogs/java/OpenJdkMigrations.scala#L218-L241

Reply:
Thanks for the pointer @eddumelendez